xmrstak/http/httpd.cpp

```diff
@@ -113,6 +113,13 @@ int httpd::req_handler(void * cls,
 		rsp = MHD_create_response_from_buffer(str.size(), (void*)str.c_str(), MHD_RESPMEM_MUST_COPY);
 		MHD_add_response_header(rsp, "Content-Type", "application/json; charset=utf-8");
 	}
+	else if(strcasecmp(url, "/prometheus") == 0)
```

Comment:
Why no to go with default "/metrics" ?

Reply:
Because it's Prometheus's related format of output. I don't have problem with metric name too. Name can be changed.

Reply:
It would require less custom settings, so I think it's better, but this is minor.